tutorial/4_hextof_workflow.ipynb

```diff
@@ -767,7 +739,7 @@
     "    constant=-1463.7, # this is time zero\n",
     "    flip_delay_axis=True, # invert the direction of the delay axis\n",
     "    columns=['bam'], # use the bam to offset the values\n",
-    "    weights=[0.001], # bam is in fs, delay in ps\n",
+    "    weights=[-0.001], # bam is in fs, delay in ps\n",
```

Comment:
We want to subtract the bam column and positive weight adds it

Reply:
Has this been tested that it does what it's intended to do?

Reply:
I have checked with Dima and this seems correct.